WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -471,11 +471,7 @@ extension WordPressAppDelegate {
             return
         }
 
-        trackDeepLink(for: url) { url in
-            DispatchQueue.main.async {
-                UniversalLinkRouter.shared.handle(url: url)
-            }
-        }
+        UniversalLinkRouter.shared.handle(url: url)
```

Comment:
The two if statements above can be removed now.

Reply:
Even better! Done.